SU-SWS/stanford_profile
# D8CORE-1789 Fix xmlsitemap 404 response

# READY FOR REVIEW

# Summary
- Fixed broken sitemap.xml path on site provision
- Fixed broken cron task that should regenerate the sitemap.

# Need Review By (Date)
- 4/8

# Urgency
- Medium

# Steps to Test
1. comment out settings in `docroot/sites/settings/xmlsitemap.settings.php`
1. `blt drupal:install`
1. verify your local site `/sitemap.xml` is functional. (the domain probably wont be correct but on acsf it will be because of the `--sitename` option during site install

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
